src/components/ActivityFeed/ActivityFeedTreasury.tsx

```
@@ -250,153 +120,385 @@ const ActivityFeedSidebar = () => {
+		setNextBurn({
+			isLoading: true,
+			value: '',
+			valueUSD: ''
+		});
 
-	useEffect(() => {
-		if (assethubApi) {
-			fetchAssetsAmount(assethubApi, assethubApiReady, network, setAssethubValues);
+		const treasuryAccount = u8aConcat(
+			'modl',
+			api.consts.treasury && api.consts.treasury.palletId ? api.consts.treasury.palletId.toU8a(true) : `${['polymesh', 'polymesh-test'].includes(network) ? 'pm' : 'pr'}/trsry`,
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)